task/04-arrays-tasks.js

```diff
@@ -249,7 +265,17 @@ function getMovingSum(arr) {
  * [ "a" ] => []
  */
 function getSecondItems(arr) {
-   throw new Error('Not implemented');
+   if (arr.length < 2) return []
+
+   const newArr = []
+   let index = 1
+
+   for (let i = 0; i < Math.floor(arr.length / 2); i++) {
+      newArr.push(arr[index])
+      index += 2
+   }
+
+   return newArr
```

Comment:
Looks like a bit overcomplicated solution, can you try to find an easier way to solve this task?

Hint: use `filter` method